### ISSUE TITLE
Verify that the photo album works with the new login flow

### ISSUE THREAD

**Issue by crtahlin** [fairDataSociety/fairdrive-apps-photoalbum#16]
As it was designed with the old one.

**Comment by tamas6** [comment]
currently not working, even cannot enter properly fairdrive

**Comment by tamas6** [comment]
comments: 
- a görli fairdrive account's TTL is around. 3 weeks max
- currently every faucet I tried takes more time to gather geth (e.g.https://goerli-faucet.pk910.de/) than the registration funding timeout. (MBP 2017 cto, used to take a 4-5 min to mine the 0.05) making a bit difficult to test for those having limited geth. 
- photoviewer (FairPhoto) in it's current state unusable : https://app.photo.dev.fairdatasociety.org/gallery
by default don't have any functionality, and after 30min still cannot see the photo i uploaded in fairdrive, only the pod I created for it but not the inner .jpg file.
- https://app.fairdrive.dev.fairdatasociety.org/drive drops out 404 when refreshin page.
- Overall experience is slower than before and unreliable
